ansible-omero-roles.md

```diff
@@ -0,0 +1,53 @@
+# OMERO component Ansible roles
+
+This is a proposal for replacing the existing [omero-server Ansible role](https://github.com/openmicroscopy/ansible-role-omero-server) with a set of omero component roles to allow the installation of decoupled OMERO components, initially server and web.
+
+
+## Rationale
+
+The existing [omero-server Ansible role](https://github.com/openmicroscopy/ansible-role-omero-server) handles most of the steps for setting up OMERO.server, with the exception of dependencies which are installed by other roles.
+This includes installing and configuring both OMERO.server and OMERO.web, and although OMERO.web can be disabled it is not possible to install OMERO.web only.
+
+Further more this role bundles `omego` which could potentially be used independently of this role.
+In addition Ansible 2.2 has a bug which means `omego` is installed into the wrong virtualenv, breaking the role.
+
+
+## Proposal
+
+Split `omero-server` into multiple Ansible roles:
+
+### omero-common
+- Create an `omero` system user
+- Create `/opt/omero` as the parent directory for all OMERO components
+- Possible: Include `omero` and `omero-web` restart handlers so other roles can restart the server or web.
+
+### omero-omego
+- Install omego into a virtualenv: `/opt/omero/omego`
+- Alternative: install `omego` globally and name this role `omego`
```

Comment:
I agree a global installation with `pip` is the easiest, especially if other people are expected to run it manually. The advantage of a virtualenv is it minimises the number of non-distribution-packaged python modules, especially if web runs from a virtualenv. Ice is probably the main exception.